guides/common/modules/ref_remote-execution-jobs-for-image-modes-hosts.adoc

```diff
@@ -0,0 +1,13 @@
+[id="remote_execution_jobs_for_image_mode_hosts_"]
+= Remote execution jobs for image mode hosts 
+
+You can use the remote execution jobs under the *Bootc* job category to manage image mode hosts:
+
+** *Bootc Action* - Script Default: Select any Bootc action from the dropdown menu, enter arguments, and specify a target.
```

Comment:
I think that the whole name of the jobs should be formatted bold. Please, fix it for all the jobs.
```suggestion
** *Bootc Action - Script Default*: Select any Bootc action from the dropdown menu, enter arguments, and specify a target.
```